### PR TITLE
mysqlclient: add recipe

### PR DESCRIPTION
**Pull request by nehaljwani**
xref: https://github.com/conda-forge/staged-recipes/pull/1008

### CODE REVIEW

File: recipes/mysqlclient/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  skip: True  # [win]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - mysql-connector-c
```
Comment (nehaljwani) [inline recipes/mysqlclient/meta.yaml]: Might need to pin, depending upon: https://github.com/lvc/upstream-tracker/issues/42

File: recipes/mysqlclient/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  skip: True  # [win]
```
Comment (nehaljwani) [inline recipes/mysqlclient/meta.yaml]: Skipping since mysql-connector-c doesn't support Windows as of now. Will try to add support later.

**Comment by nehaljwani** [comment]
cc @pmlandwehr

**Comment by nehaljwani** [comment]
I have no clue why CI failed on Windows.

File: recipes/mysqlclient/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+test:
+  imports:
+    - MySQLdb
+    - MySQLdb.constants
+    - MySQLdb.cursors
+
+about:
+  home: https://github.com/PyMySQL/mysqlclient-python
+  license: GPL 2.0
```
Comment (synapticarbors) [inline recipes/mysqlclient/meta.yaml]: license: `GPL-2.0`

Can you please also add a `license_file: LICENSE` (it looks like it should be in the source distribution)

File: recipes/mysqlclient/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  summary: 'Python interface to MySQL'
```
Comment (synapticarbors) [inline recipes/mysqlclient/meta.yaml]: Add `doc_url: https://mysqlclient.readthedocs.io/`

**Comment by synapticarbors** [comment]
I'm not sure what's going on with the appveyor error.

File: recipes/mysqlclient/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+about:
+  home: https://github.com/PyMySQL/mysqlclient-python
+  doc_url: https://mysqlclient.readthedocs.io/
+  license: GPL 2.0
+  license_file: LICENSE
+  summary: 'Python interface to MySQL'
+
+extra:
+  recipe-maintainers:
+    - pmlandwehr
```
Comment (jakirkham) [inline recipes/mysqlclient/meta.yaml]: Given some time has passed, are you still interested in helping maintain this, @pmlandwehr?

**Comment by halldc** [comment]
@nehaljwani @jakirkham Is there anything blocking this PR?

File: recipes/mysqlclient/meta.yaml (hunk 5 above)
Comment (marcelotrevisani) [reply recipes/mysqlclient/meta.yaml]: ping @pmlandwehr

File: recipes/mysqlclient/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
```
Comment (marcelotrevisani) [inline recipes/mysqlclient/meta.yaml]: Please, change that to
```
"{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
```

File: recipes/mysqlclient/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,46 @@
+{% set name = "mysqlclient" %}
+{% set version = "1.3.12" %}
```
Comment (halldc) [inline recipes/mysqlclient/meta.yaml]: Also, this should be updated to 1.3.13

**Comment by synapticarbors** [comment]
closed by https://github.com/conda-forge/staged-recipes/pull/6797

**Pull request closed without merging**